pkg/destroy/openstack/openstack.go

```diff
@@ -1822,3 +1832,68 @@ func validateCloud(opts *clientconfig.ClientOpts, logger logrus.FieldLogger) err
 
 	return networkextensions.Validate(availableExtensions)
 }
+
+func clearVIPsPorts(opts *clientconfig.ClientOpts, filter Filter, logger logrus.FieldLogger) (bool, error) {
+	logger.Debug("Clearing provided Ports for API and Ingress VIPs")
+	defer logger.Debugf("Exiting clear of provided Ports for API and Ingress VIPs")
+	conn, err := clientconfig.NewServiceClient("network", opts)
+	if err != nil {
+		logger.Error(err)
+		return false, nil
+	}
+
+	PortlistOpts := ports.ListOpts{
+		TagsAny: "dual-stack-vips-port",
```

Comment:
best practice question: should it be a constant somewhere?
so we avoid string duplication, and can easily change it in the future?

Reply:
yeah, good point. I added at [defaults/platform.go](https://github.com/openshift/installer/pull/7133/files#diff-659674db3f9f024aecde8f641878122a1ceac3febdb4ce6c8903fc3f0d2b4fcb), I didn't find other place to add